### ISSUE TITLE
Table already exists issue when installing with an existing database.

### ISSUE THREAD

**Issue by camory** [haiwen/seafile-server#188]
Hello

I have a seafiler-server, in version 6.3.4, running with MySQL. 
I tried to make a new installation of the same version of seafile-server with the existing database.
I launched the following command:
```bash
/opt/seafile/seafile-server-6.3.4/setup-seafile-mysql.sh auto \
    -n seafile \
    -i my.domain.com \
    -p 8082 \
    -e 1 \ 
    -o mariadb \
    -t 3306 \
    -r toto \
    -u toto \
    -w toto \
    -q % \
    -c ccnet-db \
    -s seafile-db \ 
    -b seahub-db
```

I got an issue saying that the table 'admin_log_adminlog' already exists.
Here are le the log of the installation:
```
verifying password of user toto ...  done

verifying user "toto" access to database ccnet-db ...  done

verifying user "toto" access to database seafile-db ...  done

verifying user "toto" access to database seahub-db ...  done

---------------------------------
This is your configuration
---------------------------------

    server name:            seafile
    server ip/domain:       my.domain.com

    seafile data dir:       /opt/seafile/seafile-data
    fileserver port:        8082

    database:               use existing
    ccnet database:         ccnet-db
    seafile database:       seafile-db
    seahub database:        seahub-db
    database user:          toto


Generating ccnet configuration ...

Generating seafile configuration ...

done
Generating seahub configuration ...

----------------------------------------
Now creating seahub database tables ...

----------------------------------------

Error: Failed to init seahub database: Table 'admin_log_adminlog' already exists
```

The new installation should not recreate the tables of seahub database as I specify to use an existing database.

**Comment by camory** [comment]
Indeed, it is a new installation but with a backup of the data and the database.
The flag -e is for existing database so I thought was for new installation with a backup of a previous installation. 
If I drop all the tables in seahub-db, the installation works perfectly and I don't get exception for existing  tables in ccnet-db and seafile-db. 
If it is not a bug what is the goal the - e flag ?

**Comment by camory** [comment]
Ok, so setting the flag `-e` to 1  is useful when we don't have admin righst to create databases. We ask the admin to create the 3 databases and Seafile populates this 3 databases with its own tables.
But we still have an inconsistent beahvior between the 3 databases. 
For `seahub-db` the install process failed because of the existing tables but it did not for the existing tables in the 2 other databases once I have droped all the tables of `seahub-db`.

In fact my question was more about the process of doing a fresh install of Seafile when you have already data. Imagine, for any reason (conrrupted file in seafile, corrupted configuration), you want to do a new installation of Seafile but you want to keep the data. What is the best process to perform it ?

**Comment by cr1st1p** [comment]
I have the same issue. Due to other reasons, mysql setup does not finish, and when program starts again, it will run again the setup-seafile-mysql script.
Problem is though strange - as @camory  correctly saw, ccnet and seafile db are both having 'IF NOT EXISTS' clauses, while seahub's db NOT.

They should be all 3 treated equally the same, and should be with 'IF NOT EXISTS' clause.
